lucene-jbossmodules-integrationtests/server-provisioning.xml

```diff
@@ -0,0 +1,53 @@
+<server-provisioning xmlns="urn:wildfly:server-provisioning:1.1" extract-schemas="true"
+                     extract-schemas-groups="org.jboss.as org.wildfly org.wildfly.core">
+    <feature-packs>
+        <feature-pack groupId="org.wildfly" artifactId="wildfly-servlet-feature-pack" version="11.0.0.Final">
+            <contents>
```

Comment:
Is there a specific reason to filter out the copyright, README, and LICENSE? I wouldn't expect those to cost much...

Reply:
No reason, was just measuring impact; I had filtering in place mainly as a workaround for other problems but turned out to be my misunderstanding. Filtering seems surprisingly cheap though so if it can trim the server we need for integration tests, why not. Possibly not worth maintenance though, but  won't really harm if someone were to rename those.